shared/wallets/confirm-form/container.js

```diff
@@ -7,21 +7,42 @@ import {connect, type TypedState} from '../../util/container'
 const mapStateToProps = (state: TypedState) => {
   const build = state.wallets.buildingPayment
   const built = state.wallets.builtPayment
-
+  const banners = (state.wallets.sentPaymentError
+    ? [
+        {
+          bannerBackground: 'HighRisk',
+          bannerText: state.wallets.sentPaymentError,
+        },
+      ]
+    : []
+  ).concat(
+    (built.banners || []).map(banner => ({
+      bannerBackground: Constants.bannerLevelToBackground(banner.level),
+      bannerText: banner.message,
+    }))
+  )
   return {
     amount: build.amount,
     assetConversion: built.worthDescription,
     assetType: build.currency,
+    banners,
     encryptedNote: build.secretNote.stringValue(),
     publicMemo: build.publicMemo.stringValue(),
+    sendFailed: !!state.wallets.sentPaymentError,
     waitingKey: Constants.sendPaymentWaitingKey,
     yourUsername: state.config.username,
   }
 }
 
 const mapDispatchToProps = (dispatch, {navigateUp}) => ({
-  onBack: () => dispatch(navigateUp()),
-  onClose: () => dispatch(navigateUp()),
+  onBack: () => {
+    dispatch(WalletsGen.createClearErrors())
```

Comment:
another pattern we have is doing this in the saga. you have a saga listener for navigate up and if its a stellar path you'd clear the error. https://github.com/keybase/client/blob/master/shared/actions/provision.js#L517

Reply:
Fixed via 03ef461